typings/src/ui/router.ts

```diff
@@ -0,0 +1,12 @@
+import { useApolloClient } from '@apollo/client';
+
+export type ApolloClient = ReturnType<typeof useApolloClient>;
```

Comment:
No need to export this. We can use the class `ApolloClient` from the library as type `ApolloClient<object>`.
`ReturnType<typeof useApolloClient>` is the same as `ApolloClient<object>`